Duplicati/GUI/Duplicati.GUI.TrayIcon/HttpServerConnection.cs

```diff
@@ -305,28 +306,25 @@ private T PerformRequest<T>(string method, string urlfragment, Dictionary<string
                         httpex.StatusDescription.IndexOf("XSRF", StringComparison.InvariantCultureIgnoreCase) >= 0)
                     {
                         hasTriedXSRF = true;
-                        string t = null;
-                        try 
-                        { 
-                            var c = httpex.Cookies[XSRF_COOKIE];
-                            if (c != null)
-                                t = c.Value;
-                        }
-                        catch
-                        {
-                        }
+                        var t = httpex.Cookies[XSRF_COOKIE]?.Value;

```

Comment:
I have tested it. Result:

httpex.Cookies[XSRF_COOKIE]?.Value
"Vu1CieBQLjIR3QqXExXkN6siIcgQeMPUbqnTts5yBcc%3D"
httpex.Cookies["test"]?.Value
null
httpex.Cookies["test"]
null